src/domain/synthetics/fees/useRebatesInfo.ts

```diff
@@ -32,11 +32,11 @@ export type RebatesInfoResult = {
   claimablePositionPriceImpactFees: RebateInfoItem[];
 };
 
-export function useRebatesInfo(chainId: number): RebatesInfoResult {
+export function useRebatesInfoRequest(chainId: number, enabled: boolean): RebatesInfoResult {
```

Comment:
why is enabled needed?

Reply:
this is ugly way to not fetch unused stuff in SyntheticsStateContextProvider

<img width="717" alt="image" src="https://github.com/gmx-io/gmx-interface/assets/144131435/bf0b9e67-58ad-4ec5-8b0c-2d15182543bf">

But I don't have a better idea for now